apps/extensions/src/content/panel/AllMediaRoutes.tsx

```
@@ -33,6 +34,10 @@ const AllMediaRoutes: FC = () => {
             path={panelRoutes.imagesShared.nestedPath}
             element={<SharedImages />}
           ></Route>
+          <Route
+            path={panelRoutes.favorites.nestedPath}
+            element={<FavoritesPage />}
+          ></Route>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)